.docker/startup.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash -e
+
+ESC_SEQ="\x1b["
+COL_RESET=$ESC_SEQ"39;49;00m"
+COL_RED=$ESC_SEQ"31;01m"
+COL_GREEN=$ESC_SEQ"32;01m"
+COL_YELLOW=$ESC_SEQ"33;01m"
+COL_BLUE=$ESC_SEQ"34;01m"
+COL_MAGENTA=$ESC_SEQ"35;01m"
+COL_CYAN=$ESC_SEQ"36;01m"
+
+if [ ! -e /var/www/html/.env ]; then
+    echo -e "░▀█▀░█▀█░▀█▀░▀█▀░▀█▀░█▀█░█░░░░░▀█▀░█▀█░█▀▀░▀█▀░█▀█░█░░░█░░\n░░█░░█░█░░█░░░█░░░█░░█▀█░█░░░░░░█░░█░█░▀▀█░░█░░█▀█░█░░░█░░\n░▀▀▀░▀░▀░▀▀▀░░▀░░▀▀▀░▀░▀░▀▀▀░░░▀▀▀░▀░▀░▀▀▀░░▀░░▀░▀░▀▀▀░▀▀▀\n"
+    echo -e $COL_YELLOW"This may take some time..."$COL_RESET
+    echo
+    cp .env.docker .env
+    composer install --no-interaction
+    php artisan key:generate
+    php artisan passport:keys
+    php artisan migrate --seed --force
```

Comment:
We have varying seeds depending on the env variable 'APP_SEEDS' 
the Default 'Usability' ones should be fine. However, there are also 'Dev' ones. Are Docker passed env variables still a thing?

Reply:
No probs, can easily be passed in as an arg.